t/022-redirect.t

```diff
@@ -218,3 +218,54 @@ GET /read
 Location: http://agentzh.org/foo?a=b&c=d
 --- response_body_like: 307 Temporary Redirect
 --- error_code: 307
+
+
+
+=== TEST 12: explicit 303
+--- config
+    location /read {
+        content_by_lua '
```

Comment:
I think it's better to change that here - we usually use `*_by_lua_block` for the new test cases :)